SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md

```diff
@@ -2,15 +2,15 @@
 
 # Tutorials and guides
 
-## Offloading example project
+## Offloading Example Project
+> **This tutorial uses the Example Project from the GDK's [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro).**</br>
 
-The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to set up a gameplay feature using offload to separate workers.
+The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to offload gameplay features to separate workers.
 
-The scenario in the `CrashBot_Gym` contains a set of turrets that attempt to attack any nearby players or AI-controlled bots. The turrets are offloaded to a server-worker called `AIWorker` and the bots are offloaded to a server-worker called `CrashBotWorker`.
+The scenario in the `CrashBot_Gym` contains a set of turrets that will attempt to attack any nearby players or AI-controlled bots. The turrets are offloaded to a worker called `AIWorker` and the bots are offloaded to a worker called `CrashBot_Worker`. This section will go through the details of what was required to enable offloading in this scenario.
```

Comment:
I don't think we need this: This section will go through the details of what was required to enable offloading in this scenario.

We already have **In this tutorial, you will go through the steps to enable offloading.**